test/e2e/setup.sh

```diff
@@ -61,13 +51,10 @@ metadata:
   name: mcad-controller-rayclusters
 subjects:
   - kind: ServiceAccount
-    name: mcad-controller-mcad
-    namespace: codeflare-system
+    name: codeflare-operator-controller-manager
```

Comment:
I'm not seeing this service account anywhere else in the code base. Where is it coming from?

Reply:
That ServiceAccount is defined here: https://github.com/project-codeflare/codeflare-operator/blob/9b2d77c378842a31b965ea4d2a5a5d18328c48c8/config/rbac/service_account.yaml#L4

But its name is changed to `codeflare-operator-controller-manager` by Kustomize global name prefixing.